src/main/java/org/apache/ibatis/scripting/xmltags/ForEachSqlNode.java

```diff
@@ -24,7 +24,7 @@
  * @author Clinton Begin
  */
 public class ForEachSqlNode implements SqlNode {
-  public static final String ITEM_PREFIX = "__frch_";
+  private static final String ITEM_PREFIX = "__frch_";
```

Comment:
This change is quite risk. Refer to the previous.

Reply:
ok, but why no comment， mybatis doc need improved.

Reply:
you are right!